X22A_validator.py

```
@@ -470,11 +487,11 @@ def ExecuteParameterSweep(omega: np.ndarray[float],
                 }
         
         # Execute MTSOL
-        exit_flag, iter_count = MTSOL_call(operating_conditions=oper,
-                                           analysis_name=ANALYSIS_NAME,
-                                           ).caller(run_viscous=True,
-                                                    generate_output=True,
-                                                    )
+        exit_flag  = MTSOL_call(operating_conditions=oper,
```

Ruff (0.8.2)

Local variable `exit_flag` is assigned to but never used

Remove assignment to unused variable `exit_flag`

(F841)